EAD/EAD-Session-11-Hibernate/EmployeePortal/src/main/java/com/metacube/training/model/JobDetail.java

```diff
@@ -0,0 +1,90 @@
+package com.metacube.training.model;
+
+import java.util.Date;
+
+import javax.persistence.*;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+@Entity
+@Table(name="job_details")
+public class JobDetail {
+
+	@Column(name="emp_code")
+	private Long emp_code;
```

Comment:
Use camelcase naming conventions for class properties. For now no need to change.